src/components/AssetHeader/QuickBuy.tsx

```
@@ -0,0 +1,243 @@
+    <Stack spacing={4}>
+      {quickBuyState.status !== 'confirming' && quickBuyState.status !== 'executing' && (
+        <>
+          <Text fontSize='md' fontWeight='bold' color='text.subtle' textAlign='center'>
+            {translate('quickBuy.title', { assetOnChain: asset?.name ?? '' })}
+          </Text>
+          <HStack spacing={2}>
+            {quickBuyAmounts.map(amount => {
+              const isSuccess =
+                quickBuyState.status === 'success' && amount === quickBuyState.amount
+              const buttonText = isSuccess ? (
+                <AnimatedCheck color='white' boxSize={6} />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)